funcbench/Makefile

```diff
@@ -28,8 +28,8 @@ resource_apply:
 resource_delete:
 	$(INFRA_CMD) gke resource delete -a ${AUTH_FILE} \
 		-v ZONE:${ZONE} -v PROJECT_ID:${PROJECT_ID} -v CLUSTER_NAME:funcbench-${PR_NUMBER} \
-		-v PR_NUMBER:${PR_NUMBER} -v GITHUB_TOKEN:${GITHUB_TOKEN} \
+		-v PR_NUMBER:${PR_NUMBER} \
 		-v GITHUB_ORG:${GITHUB_ORG} -v GITHUB_REPO:${GITHUB_REPO} \
 		-v BRANCH:${BRANCH} -v BENCH_FUNC_REGEX:${BENCH_FUNC_REGEX} \
-		-f manifests/benchmark/2_job.yaml \
+		-f manifests/benchmark/3_job.yaml \
```

Comment:
Shall we also delete the secrets here?

Reply:
thanks, i'll add that. do we even need a `resource_delete` now since we're deleting the entire cluster. will check once.

Reply:
Well, thought about it for a while. I'd say let's keep `resource_delete`, otherwise I'll be curious about why `resource_apply` doesn't have a corresponding removal task some day.